diff --git a/setup.py b/setup.py
index df44483138..0568ef1f83 100644
--- a/setup.py
+++ b/setup.py
@@ -53,6 +53,7 @@ _deps = [
     "GPUtil>=1.4.0",
     "protobuf>=3.12.2,<=3.20.1",
     "click>=7.1.2,!=8.0.0",  # latest version < 8.0 + blocked version with reported bug
+    "wandb>=v0.13.10",
 ]
 _nm_deps = [f"{'sparsezoo' if is_release else 'sparsezoo-nightly'}~={version_nm_deps}"]
 _deepsparse_deps = [
