diff --git a/src/sparseml/pytorch/optim/manager.py b/src/sparseml/pytorch/optim/manager.py
index 2d9a8c2aa8..5814a8442f 100644
--- a/src/sparseml/pytorch/optim/manager.py
+++ b/src/sparseml/pytorch/optim/manager.py
@@ -687,7 +687,12 @@ class ScheduledModifierManager(BaseManager, Modifier):
                 # check in case modifier was initialized from apply_structure
                 continue
 
-            mod.initialize(module, epoch, loggers, **kwargs)
+            mod.initialize(
+                module.module if hasattr(module, "module") else module,
+                epoch,
+                loggers,
+                **kwargs,
+            )
 
     def _finalize_modifiers(
         self,
